web-ui/src/main/resources/catalog/views/default/less/gn_search_default.less

```diff
@@ -132,6 +134,26 @@
         background-color: @gn-search-button-background-color;
         color: @gn-search-button-color;
       }
+      .btn-reset {
+        border-left-color: #fff;
```

Comment:
![image](https://user-images.githubusercontent.com/1701393/127275577-14d33dd7-0517-4f77-bc61-3b82d1a1107b.png)

Maybe not ? it depends if you have the user search widget or not

Reply:
Good catch, I will have a look